usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -68,6 +71,23 @@
         "column_names": list(AWARDS_COLUMNS),
         "tsvectors": None,
     },
+    "detached_award_procurement": {
+        "model": SourceProcurementTransaction,
+        "is_from_broker": True,
+        "source_table": "detached_award_procurement",
```

Comment:
The USAS table is `source_procurement_transaction`

Reply:
Updated